src/app_modes/topic_managment.rs

```diff
@@ -0,0 +1,365 @@
+                "Select the previous item in the active list".to_string(),
+            ],
+            [
+                input::DOWN.to_string(),
+                "Select the next item in the active list".to_string(),
+            ],
+            [
+                input::RIGHT.to_string(),
+                "Shift an element to the right if the supported topic list is active".to_string(),
+            ],
+            [
+                input::LEFT.to_string(),
+                "Shift an element to the left if the active list is active".to_string(),
+            ],
+            [
+                input::ROTATE_RIGHT.to_string(),
+                "Change the list where items are selected to the active topics list".to_string(),
+            ],
+            [
+                input::ROTATE_LEFT.to_string(),
+                "Change the list where items are selected to the supported topics list".to_string(),
+            ],
+            [
+                input::CONFIRM.to_string(),
+                "Save to config".to_string(),
+            ],
+        ]
+    }
+
+    fn get_name(&self) -> String {
+        "Topic Manager".to_string()
+    }
+}
+
+impl<B: Backend> Drawable<B> for TopicManager {
+    fn draw(&self, f: &mut Frame<B>) {
+        let title_text = vec![Spans::from(Span::styled(
+            "Topic Manager",
+            Style::default().fg(Color::Red).add_modifier(Modifier::BOLD),
+        ))];
+        let areas = Layout::default()
+            .direction(Direction::Vertical)
+            .horizontal_margin(20)
+            .constraints(
+                [
+                    Constraint::Length(3), // Title + 2 borders
+                    Constraint::Length(2),
+                    Constraint::Min(1), // Table + header + space
+                ]
+                .as_ref(),
+            )
+            .split(f.size());
+
+        let left_chunks = Layout::default()
+            .direction(Direction::Horizontal)
+            .margin(1)
+            .constraints(
+                [
+                    Constraint::Percentage(50),
+                    Constraint::Percentage(50),
+                ]
+                .as_ref(),
+            )
+            .split(areas[2]);
+        // Widget creation
+        let title = Paragraph::new(title_text)
+            .block(Block::default().borders(Borders::ALL))
+            .style(Style::default().fg(Color::White))
+            .alignment(Alignment::Center)
+            .wrap(Wrap { trim: false });
+        let items: Vec<ListItem>= self.availible_topics.items.iter().map(|i| ListItem::new(
+                format!("{} / {}", i[0], i[1])
+            )
+        ).collect();
+        // The `List` widget is then built with those items.
+        let list = List::new(items)
+            .highlight_style(Style::default().add_modifier(Modifier::BOLD))
+            .block( Block::default().title("Availible Topics").borders(Borders::ALL))
+            .highlight_symbol(">>");
```

Comment:
The initial list should have a default selection. I was confused when first opening the mode

Reply:
It does, if there are no topics active it is empty

Reply:
i meant a  selected list entry. Just select the first list entry?

Reply:
fixed